apps/docs/docs/reference/ui/atoms/Radio.mdx

```diff
@@ -1,31 +1,169 @@
 ---
 description: Radios allow users to select one option from a set.
-sidebar_custom_props: 
+sidebar_custom_props:
   image: https://vtexhelp.vtexassets.com/assets/docs/src/Radio___4bdff3538eb6a18e99c6f5a9c1043468.png
 ---
 
-import PropsSection from "@site/src/components/PropsSection/PropsSection";
+import PropsSection from '@site/src/components/PropsSection/PropsSection'
+import { TokenTable, TokenRow, TokenDivider } from '@site/src/components/Tokens'
 
 # Radio
 
 Radios allow users to select one option from a set.
 
+## Overview
+
+```tsx live
+<div style={{ display: 'flex', columnGap: '16px' }}>
+  <Radio />
+  <Radio checked />
+  <Radio disabled />
+  <Radio checked disabled />
+</div>
+```
+
 ## Import
 
+Import the component from <a href="https://www.faststore.dev/reference/ui/get-started-faststore-ui">@faststore/ui</a>
+
 ```tsx
 import { Radio } from '@faststore/ui'
 ```
 
+Import Styles
+
+```tsx
+import '@faststore/ui/src/components/atoms/Radio/styles.scss'
+```
+
 ## Usage
 
 ```tsx live
-<Radio/>
+<Radio />
 ```
 
 ## Props
 
 <PropsSection name="Radio" />
 
+## Design Tokens
```

Comment:
These `#h1`, `##h2`, `###h3` and `####h4` titles are all the same size 🥺 , can we fix it?

Reply:
Fixed!